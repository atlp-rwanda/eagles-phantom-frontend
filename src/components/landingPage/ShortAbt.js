import React from 'react';
import '../../scss/homeContent.scss';



const ShortAbt=({description})=>{
  return(
    <div className="about">
    <p>{description}</p>
    </div>
  )
}

export default ShortAbt;

