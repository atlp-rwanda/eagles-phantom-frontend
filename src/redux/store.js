import {createStore, applyMiddleware } from 'redux';
import { composeWithDevTools} from 'redux-devtools-extension';
import thunk from 'redux-thunk';
import promise from 'redux-promise-middleware';
import allReducers from './reducers/';

// const rootReducer = combineReducers({
//   count: counterReducer,
// });

// const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;

const middleware = applyMiddleware(thunk, promise)
const store = createStore(allReducers, {}, composeWithDevTools(middleware));

export default store;
