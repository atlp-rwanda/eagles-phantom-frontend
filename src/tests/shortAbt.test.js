import React from 'react';
import { Provider } from "react-redux";
import store from "../redux/store.js";
import { render } from '@testing-library/react';
import ShortAbt from '../components/landingPage/ShortAbt';

const shortAbt=()=>{
    return render(
       <Provider store={store}>
       <ShortAbt />
       </Provider>
    )
}

describe("ShortAbt",()=>{
    it("shoould render the component",()=>{
        const { asFragment }=shortAbt();
        expect(asFragment(<ShortAbt />)).toMatchSnapshot()
    });
})