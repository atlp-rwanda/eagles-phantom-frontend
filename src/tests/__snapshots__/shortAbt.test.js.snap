// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShortAbt shoould render the component 1`] = `
<DocumentFragment>
  <div
    class="about"
  >
    <p />
  </div>
</DocumentFragment>
`;
